marketplace/deployer_util/config_helper.py

```diff
@@ -23,6 +23,7 @@
 import yaml
 
 NAME_RE = re.compile(r'[a-zA-z0-9_\.\-]+$')
+SEMVER_RE = re.compile(r'^[0-9]+\.[0-9]+\..+$')
```

Comment:
Python doesn't appear to have any built-in semver modules, and this covers the main cases (e.g. partners only provide a minor version).